src/modules/dashboard/components/Extensions/ColonyPolicySelector/ColonyPolicySelector.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { MessageDescriptor } from 'react-intl';
+
+import { CustomRadioGroup, CustomRadioProps } from '~core/Fields';
+import Heading from '~core/Heading';
+
+import styles from './ColonyPolicySelector.css';
+
+export interface FormValues {
+  policy: string;
+}
+
+interface Props {
+  selectedPolicy: string;
+  title: MessageDescriptor | string;
+  options: CustomRadioProps[];
+}
+
+const ColonyPolicySelector = ({ selectedPolicy, title, options }: Props) => (
```

Comment:
Please put this (sub)component under the `Whitelist` extension's folder, not the `Extensions` folder

Reply:
A `WhitelistExtension` folder doesn't exist, I'll create it then

Reply:
I think Alicja already created it.

Make sure to check after you rebase.